src/views/datasets/buoy-telemetry/summary.vue

```diff
@@ -126,7 +126,27 @@
     </template>
   </DashboardCard>
 
-  <DashboardCard width="one-third" :height="1">
+  <DashboardCard :scenarios="scenarios" width="one-third" :height="1">
+    <template #title> Not sure what to explore? </template>
+    <template #subtitle>
+      Here are a few pre-selected scenarios for you. Just pick one and start
```

Comment:
Slight wording change if you want

```suggestion
      Here are a few pre-selected scenarios for you. Pick one to start
```